inc/entityconfig.class.php

```diff
@@ -66,6 +66,10 @@ class PluginFormcreatorEntityconfig extends CommonDBTM {
    const CONFIG_SEARCH_ISSUE_HIDDEN = 0;
    const CONFIG_SEARCH_ISSUE_VISIBLE = 1;
 
+   const CONFIG_UI_FORM_MASONRY = 'masonry';
+   const CONFIG_UI_FORM_UNIFORM_HEIGHT = 'uniform_height';
```

Comment:
Other settings use numbers

```suggestion
   const CONFIG_UI_FORM_MASONRY = 0;
   const CONFIG_UI_FORM_UNIFORM_HEIGHT = 1;
```